packages/babel-plugin-proposal-object-rest-spread/src/index.js

```diff
@@ -131,6 +131,48 @@ export default declare((api, opts) => {
     });
   }
 
+  function createObjRefExpression(path, objRef) {
+    let impureDefaultValueDeclarator;
+    let objRefExpression;
+    if (path.parentPath?.isAssignmentPattern()) {
```

Comment:
When can `parentPath` be null?

Reply:
`parentPath` can be `undefined` when [`traverse.hasType`](https://github.com/babel/babel/blob/fee4d936a3443c4d8812cac208e567e0ba8edc9d/packages/babel-traverse/src/index.js#L103) is called, but I admit I may be too paranoid here.